chaoscenter/web/src/views/Gitops/Gitops.tsx

```diff
@@ -0,0 +1,319 @@
+                                  label={
+                                    <Text font={{ variation: FontVariation.FORM_LABEL }} margin={{ top: 'medium' }}>
+                                      Access Token
+                                    </Text>
+                                  }
+                                  placeholder="Enter your Personal Acess Token"
+                                />
+                              )}
+                              {formikProps.values.authType === AuthType.SSH && (
+                                <Layout.Vertical
+                                  spacing={'small'}
+                                  flex={{ justifyContent: 'flex-start', alignItems: 'flex-start' }}
+                                >
+                                  <Button
+                                    onClick={() => {
+                                      generateSSHKeyMutation({
+                                        onCompleted: data => {
+                                          setPublicSshKey(data.generateSSHKey.publicKey);
+                                          formikProps.setFieldValue('sshPrivateKey', data.generateSSHKey.privateKey);
+                                          formikProps.setFieldValue('sshPublicKey', data.generateSSHKey.publicKey);
+                                        }
+                                      });
+                                    }}
+                                    variation={ButtonVariation.SECONDARY}
+                                    text="Generate New SSH Key"
+                                  />
+                                  <div className={css.textInputContainer}>
+                                    <Text font={{ variation: FontVariation.FORM_LABEL }} margin={{ bottom: 'xsmall' }}>
+                                      SSH Key
+                                    </Text>
+                                    <TextInput
+                                      placeholder="SSH Key"
+                                      value={sshPublicKey}
+                                      onChange={(e: FormEvent<HTMLInputElement>) => {
+                                        setPublicSshKey(e.currentTarget.value);
+                                        formikProps.setFieldValue('sshPublicKey', e.currentTarget.value);
+                                      }}
+                                      rightElement={
+                                        (
+                                          <Container
+                                            flex={{ justifyContent: 'center', alignItems: 'center' }}
+                                            height="100%"
+                                          >
+                                            <CopyButton stringToCopy={sshPublicKey} />
+                                          </Container>
+                                        ) as any
+                                      }
```

Comment:
is this working? since rightElement takes iconName only 😕

Reply:
yes it's working :)
<img width="507" alt="image" src="https://github.com/litmuschaos/litmus/assets/28081206/d869f21e-ac9b-4f3e-97dd-7448de3cafcd">